### PR TITLE
VM charts frontend

### PR DESCRIPTION
**Pull request by Gryfit**

### CODE REVIEW

**Review by MarcinAman — APPROVED** [review]

**Review by JakubSroka — APPROVED** [review]

File: src/main/webapp/app/admin/statistics/vm-statistics.service.ts [hunk 1]
```diff
@@ -9,11 +9,11 @@ import { Injectable } from '@angular/core';
 export class VmStatisticsService {
     constructor(private http: HttpClient) {}
 
-    getVmStatistics(from: Date, to: Date): Observable<VmStatistics> {
-        return this.http.get<VmStatistics>(SERVER_API_URL + 'api/statistics/vm-usage/2010-01-01/2019-02-01');
+    getVmStatistics(from: Date, to: Date): Observable<VmStatistics[]> {
+        return this.http.get<VmStatistics[]>(SERVER_API_URL + 'api/statistics/vm-usage/2010-01-01/2019-02-01');
```
Comment (MarcinAman) [inline src/main/webapp/app/admin/statistics/vm-statistics.service.ts]: Don't you feel like it should be configurable?

**Review by MarcinAman — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/admin/statistics/vm-statistics.service.ts (hunk 1 above)
Comment (Gryfit) [reply src/main/webapp/app/admin/statistics/vm-statistics.service.ts]: jeez and its your code m8

**Review by MarcinAman — APPROVED** [review]

File: src/main/webapp/app/admin/statistics/vm-statistics.service.ts (hunk 1 above)
Comment (MarcinAman) [reply src/main/webapp/app/admin/statistics/vm-statistics.service.ts]: Its your PR

**Review by MarcinAman — APPROVED** [review]

File: src/main/webapp/app/admin/statistics/vm-statistics.service.ts (hunk 1 above)
Comment (Gryfit) [reply src/main/webapp/app/admin/statistics/vm-statistics.service.ts]: My only error was approving your code but ok

**Pull request merged**